content/events/2023-2024/2023-10-25-horror-game-session.md

```diff
@@ -0,0 +1,16 @@
+---
+display_title: "Horror Game Session"
+title: "Horror Game Session"
+date: 2023-10-25T00:00:00Z
+draft: false
+layout: event
+poster: "images/event_posters/2023-2024/horror_game_session.jpg"
+poster_cover: "contain"
+poster_position: "center"
+short_description: "Join us on Wednesday for a some fun rounds of SCP: Secret Laboratory!"
+start_time: "5:00 - 8:00 PM EST"
+location: "CCSS Discord"
+location_link: "https://discord.gg/B3cdnfX3rh"
+background: "images/orientation2018-min.jpeg"
+publishdate: 2023-10-20
+---
```

Comment:
@JohnLu2004 I think we should add the link for downloading the game to the website